src/com/luxoft/java/course/shevtsov/homework/algorithm/BinarySearch.java

```diff
@@ -0,0 +1,36 @@
+package com.luxoft.java.course.shevtsov.homework.algorithm;
+
+import com.luxoft.java.course.shevtsov.homework.myUtilities.MyUtilities;
+
+public class BinarySearch {
+
+    public static Integer search(int[] ints, int searchNumber) {
+        if (ints.length > 0 && !MyUtilities.ifArrSotredByAsc(ints)) {
+            System.out.println("Invalid parameters. The Array is empty or not sorted by ascending.");
+            return null;
```

Comment:
null as return type is not good

Reply:
'-1' would be better? (if meth does not find the number or the array does not satisfy conditions)

Reply:
would be it better if method reterns '-1' if no such elemen and -2 if invalid array?

Reply:
done